src/Core/src/Platform/Windows/WrapperView.cs

```diff
@@ -33,6 +33,10 @@ public partial class WrapperView : Grid, IDisposable
 		FrameworkElement? _child;
 
 		UIElementCollection? _cachedChildren;
+		
+		// Track previous dimensions to avoid redundant UpdateClip calls
+		double _previousWidth = -1;
+		double _previousHeight = -1;
```

Comment:
remove the changes in this class, as the one clip is enough, i have shared to restrict the clip in ContentPanel